client/src/store/modules/scheduledEvent/actions.js

```diff
@@ -2,16 +2,33 @@ import * as actions from './types/actions';
 import * as mutations from './types/mutations';
 import scheduledEventService from '@/services/scheduled-event/scheduledEventsService';
 import { SET_ERROR_NOTIFICATION } from '@/store/modules/notification/types/actions';
+import * as loaderMutations from '@/store/modules/loader/types/mutations';
 
 export default {
     [actions.SET_SCHEDULED_EVENT_FILTER_VIEW]: async (
         { commit },
         scheduledEventFilterView
     ) => {
-        commit(
-            mutations.SET_SCHEDULED_EVENT_FILTER_VIEW,
-            scheduledEventFilterView
-        );
+        commit('loader/' + loaderMutations.SET_LOADING, true, { root: true });
+
+        try {
+            commit(
+                mutations.SET_SCHEDULED_EVENT_FILTER_VIEW,
+                scheduledEventFilterView
+            );
+
+            commit('loader/' + loaderMutations.SET_LOADING, false, {
+                root: true
+            });
+        } catch (error) {
```

Comment:
If token expired => 401 error => we must redirect user to login page. Use existing action from 'auth' module